lib/Service/ParticipantService.php

```diff
@@ -820,7 +815,12 @@ public function inviteEmailAddress(Room $room, string $email): Participant {
 		$attendee = new Attendee();
 		$attendee->setRoomId($room->getId());
 		$attendee->setActorType(Attendee::ACTOR_EMAILS);
-		$attendee->setActorId($email);
+		$attendee->setActorId($actorId);
+		$attendee->setInvitedCloudId($email);
```

Comment:
Unfortunate that we didn't name it `invited id` then it would be fitting. I guess it's okay to have it switch based on the actor type instead of adding yet another column?